.github/workflows/twoslash-repros.yaml

```diff
@@ -8,35 +8,28 @@ on:
     - cron:  '0 8 * * *'
   repository_dispatch:
     types: run-twoslash-repros
-  issues:
-    types:
-      - labeled
   workflow_dispatch:
     inputs:
-      bisect_issue:
-        description: Triggers a bisect request on the given issue number instead of updating repros on all issues
+      issue:
+        description: Limits run to a single issue.
+        required: false
+        type: string
+      bisect:
+        description: If set, runs a git bisect on an existing repro. Requires 'issue' to be set. Value can be revision labels (e.g. `good v4.7.3 bad main`) or `true` to infer bisect range.
         required: false
         type: string
 
 jobs:
   run:
-    if: ${{ github.repository == 'microsoft/TypeScript' && !github.event.label && !github.event.inputs.bisect_issue }}
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/setup-node@v3
-    - uses: microsoft/TypeScript-Twoslash-Repro-Action@master
-      with:
-        github-token: ${{ secrets.TS_BOT_GITHUB_TOKEN }}
-
-  bisect:
-    if: ${{ github.event.label.name == 'Bisect Repro' || github.event.inputs.bisect_issue }}
+    if: ${{ github.repository == 'microsoft/TypeScript' }}
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
       with:
-        fetch-depth: 0
+        fetch-depth: ${{ github.events.input.bisect && '0' || '1' }}
```

Comment:
Does this work? IIRC GHA expressions are kind of limited and this would be treated as a boolean (or just fail to work).

Reply:
No idea; I assume you’re right. I’ll switch it to two steps with `if`s

Reply:
Yeah, or use a previous step that sets a step output. That's pretty common for dynamic variables, e.g. https://github.com/actions/cache/blob/main/examples.md#node---npm